src/main/kotlin/com/openlattice/mechanic/upgrades/UpdateDateTimePropertyHash.kt

```diff
@@ -194,14 +218,18 @@ class UpdateDateTimePropertyHash(private val toolbox: Toolbox) : Upgrade {
                 onConflict
     }
 
-    private fun insertRehashedRowsIntoDataTableSql(): String {
+    private fun insertRehashedRowsIntoDataTableSql(partition: Int): String {
         val keyCols = DATA_TABLE_KEY_COLS.joinToString(", ")
         val insertSelectCols = (TEMP_TABLE_UNCHANGED_COLS + HASH).joinToString(", ") { it.name }
 
+        val getBatch = "WITH batch AS ( UPDATE $TEMP_TABLE_NAME set $LAST_MIGRATE = now() WHERE id in (SELECT id from $TEMP_TABLE_NAME WHERE ($LAST_MIGRATE = '-infinity') limit $BATCH_SIZE) RETURNING * )"
```

Comment:
i think the inner select should filter on partition.

Reply:
👍